protobuf/lib/src/protobuf/coded_buffer_reader.dart

```diff
@@ -46,7 +54,7 @@ class CodedBufferReader {
     byteLimit += _bufferPos;
     var oldLimit = _currentLimit;
     if ((oldLimit != -1 && byteLimit > oldLimit) || byteLimit > _sizeLimit) {
-      throw InvalidProtocolBufferException.truncatedMessage();
+      _reportTruncatedMessage(byteLimit);
```

Comment:
To make it more clear on call site, consider renaming to `_throwTruncatedMessageError` (`report` may mean it's just logging it somewhere)

Reply:
Done in #737.